<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPatternDialog</class>
 <widget class="QDialog" name="EditPatternDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>166</width>
     <height>22</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="onTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>s</string>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>3200.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="nextPattternComboBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Pattern A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pattern B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pattern C</string>
    </property>
   </item>
  </widget>
  <widget class="LEDPatternDisplay" name="patternDisplay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ColorPicker" name="colorPickerFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>On Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="onCheckBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stay On</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>236</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="redSpinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="greenSpinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="blueSpinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LEDPatternDisplay</class>
   <extends>QFrame</extends>
   <header>ledpatterndisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QFrame</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
